Kickstarter-iOS/Features/BlockingUsers/BlockUserActionSheetView.swift

```diff
@@ -0,0 +1,25 @@
+import SwiftUI
+
+@available(iOS 15.0, *)
+struct BlockUserActionSheetView: View {
+  var blockUser: () -> Void
+  var viewProfile: (() -> Void)?
+
+  var body: some View {
+    // Scott TODO: Use localized strings once translations are done
+    EmptyView()
+      .confirmationDialog("", isPresented: .constant(true), titleVisibility: .hidden) {
+        if let viewProfile = viewProfile {
+          Button("View profile") {
+            viewProfile()
```

Comment:
No need to use a trailing closure just to call one function; I think `Button("View profile", action: viewProfile)` would be more readable. This is true for the `blockUser` below, too.

Reply:
Nice. This works for view profile, but I need to control the foregroundColor of the block user label so the syntax for that will need to be something like
```
Button {
          blockUser()
        } label: {
          Text("Block this user")
            .foregroundColor(Color(.ksr_alert))
        }
```